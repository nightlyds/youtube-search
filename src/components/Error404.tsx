import React from "react";
import Footer from "./Footer";

const Error404 = (): JSX.Element => {
    return (
        <div className="error-404">
            <div className="error-404-content">
                <h2 className="error-404-title">404</h2>
                <span className="error-404-text">Page not found!</span>
            </div>
            <Footer />
        </div>
    );
};

export default Error404;
