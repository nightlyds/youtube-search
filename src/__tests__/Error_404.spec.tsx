import React from "react";
import Error404 from "../components/Error404";
import { shallow } from "enzyme";

const setUp = () => shallow(<Error404 />);

describe("Error 404 component tests", () => {
    let component: any;

    beforeEach(() => {
        component = setUp();
    });

    it("Snapshot test", () => {
        expect(component).toMatchSnapshot();
    });
});
