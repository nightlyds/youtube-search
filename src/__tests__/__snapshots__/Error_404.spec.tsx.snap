// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error 404 component tests Snapshot test 1`] = `
<div
  className="error-404"
>
  <div
    className="error-404-content"
  >
    <h2
      className="error-404-title"
    >
      404
    </h2>
    <span
      className="error-404-text"
    >
      Page not found!
    </span>
  </div>
  <Footer />
</div>
`;
